### PR TITLE
Test L2 bridge

### PR DESCRIPTION
**Pull request by fredlacs**
This PR tests the Layer 2 side of the bridge and adds deployment scripts to it.

### CODE REVIEW

File: packages/arb-bridge-peripherals/contracts/tokenbridge/arbitrum/StandardArbERC777.sol [hunk 1]
```diff
@@ -18,11 +19,11 @@ contract StandardArbERC777 is ERC777, Cloneable, IArbToken {
     }
 
     function initialize(address _bridge, address _l1Address, uint8 _decimals) external override {
-        require(address(bridge) != address(0), "ALREADY_INIT");
+        require(address(bridge) == address(0), "ALREADY_INIT");
         bridge = ArbTokenBridge(_bridge);
         l1Address = _l1Address;
 
-        require(_decimals <= 18);
+        require(_decimals <= 18, "Decimals must be less than 18");
```
Comment (DZGoldman) [inline packages/arb-bridge-peripherals/contracts/tokenbridge/arbitrum/StandardArbERC777.sol:26]: "or equal to"

File: packages/arb-bridge-peripherals/contracts/tokenbridge/arbitrum/ArbTokenBridge.sol [hunk 2]
```diff
@@ -37,16 +37,34 @@ contract ArbTokenBridge is CloneFactory {
...
+    }
+    modifier onlyToL2Token(address l1ERC20, address to) {
+        // This ensures that this method can only be called by the L2 token
+        require(to == calculateBridgedERC777Address(l1ERC20)
+            || to == calculateBridgedERC20Address(l1ERC20)
+            || to == customToken[l1ERC20], "NOT_TO_TOKEN");
+        _;
+    }
+
+    constructor(address _l1Pair) public {
```
Comment (DZGoldman) [inline packages/arb-bridge-peripherals/contracts/tokenbridge/arbitrum/ArbTokenBridge.sol:63]: why this change? (from `address(this)` to l1Pair as a param)?

Comment (fredlacs) [reply packages/arb-bridge-peripherals/contracts/tokenbridge/arbitrum/ArbTokenBridge.sol:63]: I created two subclasses, SymmetricBridge that initialises the constructor to `address(this)` and TestBridge that uses `msg.sender` so that we can mock messages coming from the core-bridge

**Comment by DZGoldman** [comment]
lgtm
